http/nixos-20-09/install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh -e
+
+packer_http=$(cat .packer_http)
+
+lsblk
+
+# Partition disk
+cat <<FDISK | fdisk /dev/sda
+n
+
+
+
+
+a
+w
+
+FDISK
+
+# Create filesystem
+mkfs.ext4 -j -L nixos /dev/sda1
+
+partprobe
```

Comment:
partprobe should come before mkfs.

Reply:
it is not needed anymore :D it was a mistake on my end!